From 682d0caaffa9bfcc2381b947fb47395fb3da98b9 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Thu, 5 Sep 2019 14:06:55 +0200
Subject: [PATCH] Fix the bogus return in seq_set_gus_bank()

Fix the compile error:
 alsa.c: In function 'seq_set_gus_bank':
 alsa.c:149:9: error: 'return' with a value, in function returning void [-Werror=return-type]
   149 |  return snd_hwdep_ioctl(hwdep, SNDRV_EMUX_IOCTL_MISC_MODE, &mode);

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 alsa.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alsa.c b/alsa.c
index e5993cb344e2..b977f44695b6 100644
--- a/alsa.c
+++ b/alsa.c
@@ -146,5 +146,5 @@ void seq_set_gus_bank(int bank)
 	mode.mode = AWE_MD_GUS_BANK;
 	mode.value = bank;
 	mode.value2 = 0;
-	return snd_hwdep_ioctl(hwdep, SNDRV_EMUX_IOCTL_MISC_MODE, &mode);
+	snd_hwdep_ioctl(hwdep, SNDRV_EMUX_IOCTL_MISC_MODE, &mode);
 }
-- 
2.16.4

